src/chapters/setup-intro-data-types/data-types.rst

```diff
@@ -201,8 +183,7 @@ be found on the
      - ``true`` and ``false``
      - Booleans in Java are NOT capitalized.
 
-Java uses 3 primitive data types to express different types of numbers, whereas
-JavaScript captures all of these in the ``number`` type.
+Java uses 3 primitive data types to express different types of numbers.
```

Comment:
We can probably remove this sentence, since we are no longer referencing the JavaScript ``number`` type.